graylog2-server/src/main/java/org/graylog2/utilities/ConfigurationMapConverter.java

```diff
@@ -51,7 +51,12 @@
                     try {
                         value = Integer.parseInt(String.valueOf(entry.getValue()));
                     } catch (NumberFormatException e) {
-                        throw new ValidationException(field, e.getMessage());
+                        // If a numeric field is optional and not provided, use null as value
+                        if ("true".equals(String.valueOf(fieldDescription.get("is_optional")))) {
```

Comment:
Not really, since `fieldDescription` values are of type `Object`. In the end we could cast the value in `is_optional` to a `String`, but that won't work if the value is actually a `Boolean`.